rosidl_typesupport_opensplice_cpp/src/identifier.cpp

```diff
@@ -18,6 +18,6 @@ namespace rosidl_typesupport_opensplice_cpp
 {
 
 ROSIDL_TYPESUPPORT_OPENSPLICE_CPP_EXPORT
-const char * typesupport_opensplice_identifier = "opensplice_static";
+const char * typesupport_identifier = "opensplice_static";
```

Comment:
should this be `opensplice_static_cpp`, like for connext [here](https://github.com/ros2/rmw_connext/pull/206/files#diff-e5c49ab610395314d92f1faa255524bcR21) ?